backend/src/install/mod.rs

```diff
@@ -564,7 +564,17 @@ pub async fn uninstall_impl(ctx: RpcContext, id: PackageId) -> Result<WithRevisi
     drop(handle);
 
     tokio::spawn(async move {
-        if let Err(e) = cleanup::uninstall(&ctx, &mut ctx.db.handle(), &id).await {
+        if let Err(e) = async {
```

Comment:
Comment/ Question: Is the reason for the double async wrap just to get the correct error out?